aem/events_mgmt_aem/src/main/java/com/adobe/aio/aem/event/management/ocd/EventProviderConfig.java

```diff
@@ -35,4 +37,9 @@
       description = "your custom documentation URL")
   String aio_provider_docs_url() default "https://developer.adobe.com/events/docs/guides/using/aem/";
 
+  @AttributeDefinition(name = "Event Delivery Format",
+      description = "Event Delivery Format: either the old legacy`adobe_io` format (used in AIO former AEM integration)"
+          + " or better the `cloud_events_v1` open specification (see https://github.com/cloudevents/spec/blob/v1.0/spec.md)")
+  String event_delivery_format() default DELIVERY_FORMAT_CLOUD_EVENTS_V1;
```

Comment:
Just wondering if someone upgrades their package, then the delivery format changes could break their integrations right?

Reply:
yes I will add a note in the release changelog see https://github.com/adobe/aio-lib-java/releases

Reply:
It could be still be a surprise for many customers, no?